robottelo/common/constants.py

```diff
@@ -7,12 +7,17 @@
 
 FOREMAN_PROVIDERS = {
     'libvirt': 'Libvirt',
-    'ovirt': 'Ovirt',
+    'rhev': 'RHEV',
     'ec2': 'EC2',
-    'vmware': 'Vmware',
-    'openstack': 'Openstack',
+    'vmware': 'VMWare',
+    'openstack': 'RHEL OpenStack Platform',
     'rackspace': 'Rackspace',
-    'gce': 'GCE'}
+    'google': 'Google',
+    'docker': 'Docker',
+}
+
+LIBVERT_RESOURCE_URL = 'qemu+tcp://%s:16509/system'
+DOCKER_RESOURCE_URL = 'http://%s:2375'
```

Comment:
Nope, ```get_internal_docker_url``` return static address ('localhost'), but here we can pass any URL that we like (e.g. server address). Definitely, we can improve mentioned functions and if you like I can create github issue for that purpose